planner/utils/google_utils.py

```diff
@@ -0,0 +1,79 @@
+"""
+Functions for getting attributes about points of interest.
+Extracts data from POI locations (elevation, safety, etc.)
+
+Uses: Google Place Search
+
+Uses the python-google-places wrapper:
+https://github.com/slimkrazy/python-google-places
+"""
+
+from googleplaces import GooglePlaces, types
+from configparser import ConfigParser
+import psycopg2
+import json
+
+class GoogleHelper:
+	"""
+	Helper class for accessing the google API
+	"""
+	def __init__(self, gmaps_api_key=None, conn=None):
+		import db_conn
+		if gmaps_api_key is None:
+			config = ConfigParser()
+			with open('config.json') as f:
+			    config = json.load(f)
+			gmaps_api_key = config["gmapsApiKey"]
+		if conn is None:
+			conn = db_conn.connPool.getconn()
```

Comment:
This leads to connections being eventually exhausted from the connection pool. Should return connection when done.

Consider implementing `__enter__` and `__exit__` in order to support the `with GoogleHelper() as ghelper:` syntax and cleans up after itself.